docs/integrations/public-ip.md

```diff
@@ -0,0 +1,25 @@
+# Pipes' Public IP Addresses
+
+## **Overview**
+This document provides a list of public IPs used by Pipes for outbound requests. These IPs are intended for end-users to configure whitelisting, firewall rules, and integrations with external services.
+
+## **Public IPs**
+#### **1. Load Balancer IPs**
+The following IPs are used by Pipes' load balancers. Please ensure these are whitelisted for access to our services:
+
+| **Load Balancer Name**       | **Static IP Address**     | **Ports**     |
```

Comment:
General comments:

- Should these be /32 CIDRs?
- Please line up the markdown across the row cols

Reply:
General practice is to use /32 so updated the doc with the same.